packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -76,7 +76,7 @@ function addHost (span, topology) {
 
 function wrapCallback (tracer, span, done, cursor) {
   return tracer.scope().bind((err, res) => {
-    if (err) {
+    if (err instanceof Error) {
```

Comment:
what else can it be?

Reply:
This change was made of another change that ended up not making it for this PR. Removed the check.